worker.js

```diff
@@ -55,24 +54,26 @@ self.onmessage = function (ev) {
   let n = -1;
 
   while (++n < times) {
+    const lru = caches[id](num);
```

Comment:
This isn't needed; there's no change to how it runs.

Reply:
If you don't do that, subsequent `set` are in fact evicts since the cache is already full, no?